Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -192,6 +223,115 @@ lemma rnDeriv_gaussianReal (μ : ℝ) (v : ℝ≥0) :
   · rw [gaussianReal_of_var_ne_zero _ hv]
     exact Measure.rnDeriv_withDensity _ (measurable_gaussianPdf μ v)
 
+section Transformations
+
+variable {μ : ℝ} {v : ℝ≥0}
+
+lemma _root_.MeasurableEmbedding.gaussianReal_comap_apply (hv : v ≠ 0)
+    {f : ℝ → ℝ} (hf : MeasurableEmbedding f)
+    {f' : ℝ → ℝ} (h_deriv : ∀ x, HasDerivAt f (f' x) x) {s : Set ℝ} (hs : MeasurableSet s) :
+    (gaussianReal μ v).comap f s
+      = ENNReal.ofReal (∫ x in s, |f' x| * gaussianPdfReal μ v (f x)) := by
+  rw [gaussianReal_of_var_ne_zero _ hv, gaussianPdf_def]
+  exact hf.withDensity_ofReal_comap_apply_eq_integral_abs_deriv_mul' hs h_deriv
+    (ae_of_all _ (gaussianPdfReal_nonneg _ _)) (integrable_gaussianPdfReal _ _)
+
+lemma _root_.MeasurableEquiv.gaussianReal_map_symm_apply (hv : v ≠ 0) (f : ℝ ≃ᵐ ℝ) {f' : ℝ → ℝ}
+    (h_deriv : ∀ x, HasDerivAt f (f' x) x) {s : Set ℝ} (hs : MeasurableSet s) :
+    (gaussianReal μ v).map f.symm s
+      = ENNReal.ofReal (∫ x in s, |f' x| * gaussianPdfReal μ v (f x)) := by
+  rw [gaussianReal_of_var_ne_zero _ hv, gaussianPdf_def]
+  exact f.withDensity_ofReal_map_symm_apply_eq_integral_abs_deriv_mul' hs h_deriv
+    (ae_of_all _ (gaussianPdfReal_nonneg _ _)) (integrable_gaussianPdfReal _ _)
+
+/-- The map of a Gaussian distribution by addition of a constant is a Gaussian. -/
+lemma gaussianReal_map_add_const (y : ℝ) :
+    (gaussianReal μ v).map (· + y) = gaussianReal (μ + y) v := by
```

Comment:
I think it would be good to add the stupid symmetric versions of this lemma and the one below (`const_add` and `mul_const`)